Source/WebCore/rendering/TextBoxPainter.cpp

```diff
@@ -1055,8 +1055,25 @@ void TextBoxPainter<TextBoxPath>::paintPlatformDocumentMarkers()
         });
     }
 
-    for (auto& markedText : MarkedText::subdivide(markedTexts, MarkedText::OverlapStrategy::Frontmost))
-        paintPlatformDocumentMarker(markedText);
+    auto transparentContentMarkedTexts = MarkedText::collectForDraggedAndTransparentContent(DocumentMarker::Type::TransparentContent, m_renderer, m_selectableRange);
+
+    // Ensure the transparent content marked texts go first in the vector, so that they take precedence over
+    // the other marked texts when being subdivided so that they do not get painted.
+    Vector<MarkedText> allMarkedTexts;
+    allMarkedTexts.appendVector(transparentContentMarkedTexts);
+    allMarkedTexts.appendVector(markedTexts);
```

Comment:
Instead of making a new vector, consider:

```
markedTexts.insertVector(0, transparentContentMarkedTexts);
```

Reply:
I considered this, I did it this way for less mutability of variables which makes things easier to read imo. I can change it if you feel strongly though?

Reply:
Yes I see your point. I suggested this to avoid an extra vector copy, but I don't have a strong opinion.